tests/jax/models/test_simple_nn.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Sequence
+
+import jax
+import jax.numpy as jnp
+import pytest
+from flax import nnx
+from infra import BaseModelTester, ComparisonConfig, Workload
+
+
+class SimpleNN(nnx.Module):
+    def __call__(
+        self, act: jax.Array, w0: jax.Array, b0: jax.Array, w1: jax.Array, b1: jax.Array
+    ) -> jax.Array:
+        x = jnp.matmul(act, w0) + b0
+        x = jnp.matmul(x, w1) + b1
+        return x
+
+
+class SimpleNNTester(BaseModelTester):
+    @staticmethod
+    def _get_model() -> nnx.Module:
+        return SimpleNN()
+
+    @staticmethod
+    def _get_model_inputs() -> Sequence[jax.Array]:
+        act_shape, w0_shape, b0_shape, w1_shape, b1_shape = (
+            (32, 784),
+            (784, 128),
+            (1, 128),
+            (128, 128),
+            (1, 128),
+        )
+
+        act = jax.numpy.ones(act_shape)
+        w0 = jax.numpy.ones(w0_shape)
+        b0 = jax.numpy.ones(b0_shape)
+        w1 = jax.numpy.ones(w1_shape)
+        b1 = jax.numpy.zeros(b1_shape)
+
+        return [act, w0, b0, w1, b1]
+
+    # @override
+    @staticmethod
+    def _get_model_as_workload() -> Workload:
+        return Workload(SimpleNNTester._get_model(), SimpleNNTester._get_model_inputs())
+
+    # @override
+    @staticmethod
+    def _get_model_forward_pass_method_name() -> str:
+        return "__call__"
```

Comment:
Do we need to override these? Seems like this will be a default behavior. `_get_model_forward_pass_method_name` already is, and `_get_model_as_workload` is unimplemented in the base but should probably be implemented in this way.